Controller/PlatformUIController.php

```diff
@@ -10,50 +10,16 @@
 namespace EzSystems\PlatformUIBundle\Controller;
 
 use eZ\Bundle\EzPublishCoreBundle\Controller;
-use Symfony\Component\HttpFoundation\Session\SessionInterface;
-use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+use EzSystems\PlatformUIBundle\ApplicationConfig\Aggregator;
 
 class PlatformUIController extends Controller
 {
-    /**
-     * @var \Symfony\Component\HttpFoundation\Session\SessionInterface
-     */
-    private $session;
-
-    /**
-     * @var \Symfony\Component\Security\Csrf\CsrfTokenManagerInterface
-     */
-    private $csrfTokenManager;
+    /** @var Aggregator */
+    private $parametersAggregator;
 
-    /**
-     * @var string
-     */
-    private $csrfTokenIntention;
-
-    /**
-     * The configured anonymous user id.
-     *
-     * @var int
-     */
-    private $anonymousUserId;
-
-    /**
-     * @var array
-     */
-    protected $countriesInfo;
-
-    public function __construct(
-        array $countriesInfo,
-        SessionInterface $session,
-        CsrfTokenManagerInterface $csrfTokenManager,
-        $restIntention = 'rest',
-        $anonymousUserId = 10
-    ) {
-        $this->session = $session;
-        $this->csrfTokenManager = $csrfTokenManager;
-        $this->csrfTokenIntention = $restIntention;
-        $this->anonymousUserId = $anonymousUserId;
-        $this->countriesInfo = $countriesInfo;
+    public function __construct(Aggregator $parametersAggregator)
```

Comment:
Assuming you take my first comment into account (aggregator implementing `Provider` interface), you should type hint against the interface instead.